### ISSUE TITLE
FIA_SOS.2 renewal after deprecation

### ISSUE THREAD

**Issue by woodbe** [DSC-iTC/cPP#192]
Aspect of possible renewal in context of deprecation, key word life cycle is missing here completely

**Comment by woodbe** [comment]
I think the point here is that we don't specify anything about generation of new credentials anywhere. That is a good point, I don't think there is anything about it, the closest thing I can see is FIA_AFL_EXT.1.3 where it talks about preventing attempts, but there isn't anything about changing the auth data.

On one hand I could see this just being closed since any time auth data is generated it needs to meet this, so it shouldn't actually matter if it is initial data or renewal data, but it does make me think that maybe it would be useful to ensure that old data can't be used in some manner? This could be as simple as ensuring that new auth data is not the same as existing auth data or it could be auth data history (I would keep it simple at this point and just ensure it didn't somehow randomly choose the same thing).

I am not sure where to put this though. Since this doesn't talk about generating the data itself, only the strength requirements, I'm not sure that putting life cycle here (or at least "not a duplicate" is applicable, but we don't have another SFR that would cover this.

I this this one could be tabled for v3 and come back to thinking about it then.

**Comment by slpotte** [comment]
I agree with Brian that auth data regeneration should be deferred and go in the parking lot.
